packages/site-kit/src/lib/docs/DocsOnThisPage.svelte

```diff
@@ -238,7 +238,7 @@
 		position: fixed;
 		padding: var(--sk-page-padding-top) var(--sk-page-padding-side) 0 0;
 		width: min(280px, calc(var(--sidebar-width) - var(--sk-page-padding-side)));
-		height: calc(100vh - var(--sk-nav-height) - var(--sk-page-padding-top));
+		height: max-content;
```

Comment:
```suggestion
		max-height: calc(100vh - var(--sk-nav-height) - var(--sk-page-padding-top));
```

Reply:
This fixes the issue of the aside not scrolling when the page height is shorter than the aside's contents and allows for scrolling the main page in the blank space.

Added grey border to element here to demonstrate.

https://github.com/sveltejs/site-kit/assets/4416345/bf3a8b23-0619-43c6-bc56-abf54d4cc2a6

Reply:
@mkwsnyder pardon the bump, mind taking a look at the proposed fix?